operator/apis/loki/v1/lokistack_types.go

```diff
@@ -803,6 +811,40 @@ type LokiStackSpec struct {
 	Tenants *TenantsSpec `json:"tenants,omitempty"`
 }
 
+type ReplicationSpec struct {
+	// Factor defines the policy for log stream replication.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum:=1
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:number",displayName="Replication Factor"
+	Factor int32 `json:"factor,omitempty"`
+
+	// Zone is the key that defines a topology in the Nodes' labels.
+	//
+	// +required
```

Comment:
One thing that it's not clear to me. And I'm not sure if it was covered in the EP, if so please point me to the discussion. 
If we make this field required aren't we forcing users that move from the old API `ReplicationFactor` to the new one `Replication.Factor` to also have to provide `Replication.Zones`? Is this the goal?